PLUGINS.md

```diff
@@ -154,6 +155,18 @@ A plugin to analyze and inspect Prow CI job results
 
 See [plugins/prow-job/README.md](plugins/prow-job/README.md) for detailed documentation.
 
+### Release Plugin
+
+A plugin to manage release configurations
+
+**Commands:**
+- **`/release:find-main-periodics` `[path] [--verify-release=VERSION]`** - Find test definitions with periodic scheduling in main/master branch configurations
+- **`/release:find-missing-variant-periodics` `<from-release> <to-release> [path]`** - Find periodic configurations missing for a target release version
+- **`/release:migrate-main-periodics-to-variant-config` `<target-release> [path|--filter=<json>] [--confirm-each-test]`** - Move periodic test definitions from main/master branch configs to dedicated __periodics.yaml files
```

Comment:
Same as [above](https://github.com/openshift-eng/ai-helpers/pull/75/files#r2490523388). I don't think this is actually the right thing to do (if it was, then [ci-tools#4614](https://github.com/openshift/ci-tools/pull/4614) would be the correct fix back then).